tailcfg/tailcfg.go

```diff
@@ -2242,8 +2243,8 @@ const (
 	// NodeAttrDisableWebClient disables using the web client.
 	NodeAttrDisableWebClient NodeCapability = "disable-web-client"
 
-	// NodeAttrExitDstNetworkFlowLog enables exit node destinations in network flow logs.
-	NodeAttrExitDstNetworkFlowLog NodeCapability = "exit-dst-network-flow-log"
+	// NodeAttrLogExitFlows enables exit flow logging.
```

Comment:
Can we keep the more descriptive doc line even though we've shortened the cap name? "...enables exit node destinations in network flow logs." is very clear for what it does.

Reply:
Agreed